b2b-sample/package-lock.json

```
@@ -56,15 +56,20 @@
         "jest": "28.1.1",
         "jest-environment-jsdom": "28.1.1",
         "less": "^4.1.3",
-        "next-auth": "^4.16.4",
+        "next-auth": "^4.24.7",
         "next-with-less": "^2.0.5",
         "nx": "15.1.1",
         "react-final-form": "^6.5.9",
         "react-test-renderer": "18.2.0",
         "rsuite": "^5.22.2",
         "ts-jest": "28.0.5",
         "ts-node": "10.9.1",
+        "typescript": "^4.9.3",
         "url-loader": "^4.1.1"
+      },
+      "engines": {
+        "node": ">=16",
+        "pnpm": ">=8"
```

Can we update as follows, because in readme we have said 16.16 and above are supported? Otherwise someone with v16.1 won't get any warnings.

```suggestion
        "node": ">=16.16.0",
        "pnpm": ">=8.11.0"
```